packages/language-server/src/util/toLS.ts

```diff
@@ -93,8 +93,13 @@ export function documentHighlight(
 	}))
 }
 
-export function documentSelector(meta: core.MetaRegistry): ls.DocumentSelector {
-	const ans: ls.DocumentSelector = meta.getLanguages().map((id) => ({ language: id }))
+export function documentSelector(
+	meta: core.MetaRegistry,
+	disabledLanguages?: string[],
```

Comment:
(nit) can `disabledLanguages` be part of an options bag instead? so we call `toLS.documentSelector(meta, { disabledLanguages })` instead. imo makes it clearer what the second argument is and we could potentially add more options in the future.